mapbuilder/handlers/jinja.py

```diff
@@ -97,6 +98,9 @@ def to_text_buffer(geometry, label: str, color: str, adapt_to_length=True):
 
     _render_polygon(lines, [buffer], color)
 
+    for i in range(len(lines)):
+        lines[i] = lines[i].replace("COLOR", "\nCOLOR")
```

Comment:
This looks like the wrong position to fix this. Are you sure you are using the Jinja templating language correctly?